OPUNetGameSelectWnd.cpp

```diff
@@ -494,10 +482,7 @@ void OPUNetGameSelectWnd::OnTimer()
 
 void OPUNetGameSelectWnd::OnReceive(Packet &packet)
 {
-	int numGames;
-	int i;
-	HostedGameInfo* hostedGameInfo;
-	LVITEM item;
+	int gameCount;

```

Comment:
This variable is tougher to remove since it would be intialized in the switch statement.

Reply:
Indeed. The method needs to be refactored and split.

Reply:
Just added the bracket for now to get the variable declaration at point of use.